cypress/support/fragments/quickMarcEditor.js

```diff
@@ -453,6 +456,15 @@ export default {
     cy.do(unlinkModal.find(unlinkButtonInsideModal).click());
   },
 
+  checkUnlinkModal(rowIndex, text) {
+    cy.do(QuickMarcEditorRow({ index: rowIndex }).find(unlinkIconButton).click());
+    cy.expect(unlinkModal.exists());
+    cy.expect(unlinkButtonInsideModal.exists());
+    cy.expect(cancelUnlinkButtonInsideModal.exists());
+    cy.expect(unlinkModal.has({ content: including(text) }));
```

Comment:
Move all above cy.expects checks to array -> cy.expect([])

Reply:
fixed